### DISCUSSION TITLE
Windows controls fail in OMEdit for Windows 10

### DISCUSSION THREAD

**Discussion by mpradovelasco** [OpenModelica/OpenModelica#10774] [Q&A]
Hi, 

The maximization, miminization and close buttoms at the top - right of windows do not appear in OMEdit (Modeling and Plotting views) (1.21 and latest 1.22-dev) for the subWindows view (Windows 10) when the window is maximized. When it is not maximized, the controls appear but they do not operate properly (do not react to the mouse click). 

They appear and operate correctly in Windows 11. 

Thanks in advance.

**Comment by casella** [comment]
@adeas31 any idea?

**Comment by adeas31** [comment]
See #6387. I have no idea if this is because of our code or Qt or Windows. I remember I debugged this long ago but couldn't find any solution.

As @mpradovelasco said that it works in Windows 11 then perhaps its some Windows 10 rendering issue. I also noticed that in Windows 10 changing screen resolution displays the controls correctly.

**Comment by mpradovelasco** [comment]
Hello, 
I have tested again, changing to a screen without scaling and it works better (Windows 10). It is not a great problem (of course, in Windows 11 works fine), but I asked it in case there was any trick to overcome it. 
Thanks for your help.
